.travis.yml

```diff
@@ -57,8 +61,8 @@ after_success:
     - sudo mv .coverage .coverage.orig || true
     - sudo mv tests/.coverage .coverage.tox || true
     - sudo coverage combine --append .
-    - coveralls
-    - codecov
+    - test "${use_coveralls}" == "true" && coveralls || true
+    - test "${use_codecov}" == "true" &&  codecov || true
```

Comment:
I don't understand why the last `|| true` are needed. Other than that, it looks fine to me.

Reply:
Travis aborts if a command returns a non-zero exit code and if one of those environment variables is not set to `true`, the return code will not be zero. In this case, Travis should not consider this an error. the `|| true` makes sure that the return code will always be `0`.